src/index.ts

```diff
@@ -142,4 +143,12 @@ export function submitEmail(fn: (target, widget, email) => any): void {
   widgets().submitEmail(fn);
 }
 
-if (window) asyncLoad();
+export function dropCookie():void {
```

Comment:
In retrospect "drop" sounds too much like "delete"
```suggestion
export function pushCookie():void {
```